okhttp/src/test/java/io/grpc/transport/okhttp/OkHttpClientTransportTest.java

```diff
@@ -518,8 +519,8 @@ synchronized void throwIoExceptionForNextFrame() {
       notifyAll();
     }
 
-    synchronized void finishNextFrame(boolean result) {
-      nextFrameResult = result;
+    synchronized void failNextFrame() {
```

Comment:
failNextFrame seems like it would be the same as throwIoExcetpionForNextFrame, just based on the name.

Maybe "causeEndOfStream" or "nextFrameAtEndOfStream" or similar?